server/lib/utils/connstr-mysql.js

```diff
@@ -0,0 +1,33 @@
+import URI from "urijs";
+
+export function parseConnectionConfig(settings) {
```

Comment:
if we have a file exporting one function I would like it to be named after the function - so in that case it would be `parse-connection-config.js`.
Also - the name of the file suggests it's related to the `mysql` only? If so it should be done inside the adapter.